File: SpringProjects/SpringFinalProject/Frontend/SpringBootFinalProject/src/app/DTOs/channelling-dto.ts
export class ChannellingDTO{

  id: number;
  hospital: number;
  doctor: number;
  day: string;
  statTime: string;
  endTime: string;
  room: string;
  price: number;
  status: string;

  constructor(private hos, private doc, private d, private st, private et, private rm, private p){
    this.hospital = hos;
    this.doctor = doc;
    this.day = d;
    this.statTime = st;
    this.endTime = et;
    this.room = rm;
    this.price = p;
  }

}

File: SpringProjects/SpringFinalProject/Backend/ChannellingService/src/main/java/e_channelling/channelling_service/dto/ChannellingDto.java
package e_channelling.channelling_service.dto;

import e_channelling.channelling_service.commonModels.Category;
import e_channelling.channelling_service.commonModels.Doctor;
import e_channelling.channelling_service.commonModels.Hospital;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.validation.constraints.NotNull;
import java.time.Instant;
import java.util.Date;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ChannellingDto {

    private Integer id;

    private Hospital hospital;

    private Doctor doctor;

    private Category[] categories;

    private String room;

    private Double price;

    private Date startTime;

    private Date endTime;

    private String status;

}
